guacamole-common-js/src/main/webapp/modules/AudioRecorder.js

```diff
@@ -183,14 +183,19 @@ Guacamole.RawAudioRecorder = function RawAudioRecorder(stream, mimetype) {
 
     /**
      * A function which directly invokes the browser's implementation of
```

Comment:
This JSDoc comment is no longer valid now that `var getUserMedia` is gone.

Reply:
Removed.